<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1031</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sensitivity Analysis</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item row="0" column="1" colspan="2">
     <layout class="QGridLayout" name="grid"/>
    </item>
    <item row="0" column="0">
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Component</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="component_dropdown"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Variable 1</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="var1_dropdown"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Variable 2</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="var2_dropdown"/>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QPushButton" name="load_button">
        <property name="text">
         <string>Calculate Sensitivity</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>16777214</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>16777214</width>
     <height>24</height>
    </size>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuMode">
    <property name="title">
     <string>Mode</string>
    </property>
    <addaction name="action_prediction"/>
    <addaction name="separator"/>
    <addaction name="action_newoptimisation"/>
    <addaction name="action_optimisation"/>
    <addaction name="separator"/>
    <addaction name="action_developer"/>
   </widget>
   <widget class="QMenu" name="menuFile_2">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuExit">
    <property name="title">
     <string>Exit</string>
    </property>
   </widget>
   <addaction name="menuFile_2"/>
   <addaction name="menuMode"/>
   <addaction name="menuExit"/>
  </widget>
  <action name="action_prediction">
   <property name="text">
    <string>Prediction</string>
   </property>
  </action>
  <action name="action_optimisation">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Optimisation Workspace</string>
   </property>
  </action>
  <action name="action_sensitivity">
   <property name="text">
    <string>Sensitivity Analysis</string>
   </property>
  </action>
  <action name="action_developer">
   <property name="text">
    <string>Developer</string>
   </property>
  </action>
  <action name="action_newoptimisation">
   <property name="text">
    <string>New Optimisation</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
